core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -111,8 +112,12 @@ object FormattedBy extends Enum[FormattedBy] {
   final case object Custom extends FormattedBy {
     override def asString: String = "CUSTOM"
   }
+
+  final case object Cromwell extends FormattedBy {
```

Comment:
Nit: It's interesting that we have both FormattedBy and AppType, even though they are the same strings. IMO it would be simpler if there was just a single enum.

Reply:
Yes. I am not sure in which case one would be different from another. Maybe the IA team can answer this. I didn't have enough knowledge on it's background hence didn't collapse it into single enum.

Reply:
In the case that the formattedBy is "GCE" there will not be an AppType, so I do not think we can collapse these values just yet.